bundle.yaml

```
@@ -1,33 +1,41 @@
-  pytorch-operator:          { charm: cs:pytorch-operator-53,         scale: 1 }
-  spark:                     { charm: cs:~spark-charmers/spark-2,     scale: 1 }
-  seldon-controller-manager: { charm: cs:seldon-core-50,              scale: 1 }
-  tfjob-operator:            { charm: cs:tfjob-operator-1,            scale: 1 }
+  admission-webhook:             { charm: admission-webhook,       channel: latest/edge, scale: 1 }
+  argo-controller:               { charm: argo-controller,         channel: latest/edge, scale: 1 }
+  argo-server:                   { charm: argo-server,             channel: latest/edge, scale: 1 }
+  dex-auth:                      { charm: dex-auth,                channel: latest/edge, scale: 1 }
+  envoy:                         { charm: envoy,                   channel: latest/edge, scale: 1 }
+  istio-ingressgateway-operator: { charm: istio-gateway,           channel: latest/edge, scale: 1, trust: true, options: { kind: ingress } }
+  istio-pilot:                   { charm: istio-pilot,             channel: latest/edge, scale: 1, trust: true, options: { default-gateways: "kubeflow-gateway" } }
+  jupyter-controller:            { charm: jupyter-controller,      channel: latest/edge, scale: 1 }
+  jupyter-ui:                    { charm: jupyter-ui,              channel: latest/edge, scale: 1 }
+  katib-controller:              { charm: katib-controller,        channel: latest/edge, scale: 1 }
+  katib-db:                      { charm: cs:~charmed-osm/mariadb-k8s-35, scale: 1, options: { database: katib } }
+  katib-db-manager:              { charm: katib-db-manager,        channel: latest/edge, scale: 1 }
+  katib-ui:                      { charm: katib-ui,                channel: latest/edge, scale: 1 }
+  kfp-api:                       { charm: kfp-api,                 channel: latest/edge, scale: 1 }
+  kfp-db:                        { charm: cs:~charmed-osm/mariadb-k8s-35, scale: 1, options: { database: mlpipeline } }
+  kfp-persistence:               { charm: kfp-persistence,         channel: latest/edge, scale: 1 }
+  kfp-profile-controller:        { charm: kfp-profile-controller,  channel: latest/edge, scale: 1 }
+  kfp-schedwf:                   { charm: kfp-schedwf,             channel: latest/edge, scale: 1 }
+  kfp-ui:                        { charm: kfp-ui,                  channel: latest/edge, scale: 1 }
+  kfp-viewer:                    { charm: kfp-viewer,              channel: latest/edge, scale: 1 }
+  kfp-viz:                       { charm: kfp-viz,                 channel: latest/edge, scale: 1 }
+  kubeflow-dashboard:            { charm: kubeflow-dashboard,      channel: latest/edge, scale: 1 }
+  kubeflow-profiles:             { charm: kubeflow-profiles,       channel: latest/edge, scale: 1 }
+  kubeflow-volumes:              { charm: kubeflow-volumes,        channel: latest/edge, scale: 1 }
+  metacontroller-operator:       { charm: metacontroller-operator, channel: latest/edge, scale: 1 }
```

Fixes "metacontroller needs trust" error
```suggestion
  metacontroller-operator:       { charm: metacontroller-operator, channel: latest/edge, scale: 1, trust: true }
```